### ISSUE TITLE
NVDA fails to read correct page numbers in the pages option (thumbnail images) in Adobe Reader 8

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#2866]
**Reported by bramhini on 2012-12-12 10:14**
while using NVDA to read documents in the Adobe Reader 8, it fails to recognize the proper page numbers. sometimes it reads 4 as 3 and 5 as 2.and it fails to read the last page number of the last thumbnail always.

**Comment by ehollig** [comment]
Closing because of reasons mentioned in https://github.com/nvaccessAuto

1. Exact steps to reproduce have not been provided.
2. Adobe Reader 8 is now 4 versions behind the latest version (Adobe Reader DC).
